Mu2eG4/src/Mu2eG4MT_module.cc

```diff
@@ -105,6 +105,8 @@ namespace mu2e {
     Mu2eG4TrajectoryControl trajectoryControl_;
     Mu2eG4Inputs multiStagePars_;
 
+    unsigned simStage_;
```

Comment:
I encouraging the practice of initializing within the declaration where that's possible, rather than in the c'tor initializer list.  Your call.

Reply:
Multiple class member right before and after simStage_ have to be initialized in the constructor list.   I think maintaining the style makes sense in this case.  Makes it easier to see that no name is missing in the initializer list.

Reply:
As I said, it's your call.  I think of initializing in the declaration as a way of minimizing cruft in the initializer list.